spencer_caldwell/data/content.txt

```diff
@@ -0,0 +1 @@
+some content
```

Comment:
You can create an empty file called `.gitkeep` to put in any empty directories that you would like git to track.  They will be hidden, but they count as content so git will track them.  Nifty, right?